components/page-header/nz-page-header.module.ts

```diff
@@ -31,12 +31,13 @@ const NzPageHeaderCells = [
   NzPageHeaderTagDirective,
   NzPageHeaderExtraDirective,
   NzPageHeaderFooterDirective,
-  NzPageHeaderBreadcrumbDirective
+  NzPageHeaderBreadcrumbDirective,
+  NzPageHeaderAvatarDirective
 ];
 
 @NgModule({
-  imports: [CommonModule, NzAddOnModule, NzIconModule, NzDividerModule],
-  exports: [NzPageHeaderComponent, ...NzPageHeaderCells],
-  declarations: [NzPageHeaderComponent, ...NzPageHeaderCells]
```

Comment:
This breaks the page header for me for some reason:
![image](https://user-images.githubusercontent.com/36156020/66913515-73430280-f01d-11e9-99df-b27742ed46cc.png)

8.3.0:
![image](https://user-images.githubusercontent.com/36156020/66913655-c3ba6000-f01d-11e9-8749-b57cbfb12e8d.png)

Reply:
Thank, we have found the breakchange in styles, we will fix it in the next version.

Reply:
Thanks!